tests/test_wire/test_rewire.py

```diff
@@ -11,3 +11,27 @@ def test_anonymous_bits_rewiring_warning(first, second, caplog):
     x = m.as_bits(x)
     x @= second
     assert has_warning(caplog)
+
+
+@pytest.fixture
+def _setup_and_teardown():
```

Comment:
Once #1090 is merged, we can use `magma.testing.utils.with_config` which does exactly this.